pvlib/ivtools.py

```diff
@@ -262,6 +262,142 @@ def fit_sde_sandia(voltage, current, v_oc=None, i_sc=None, v_mp_i_mp=None,
                                      v_oc)
 
 
+def fit_sdm_desoto(v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc,
+                   cells_in_series, EgRef=1.121, dEgdT=-0.0002677,
+                   temp_ref=25, irrad_ref=1000):
+    """
+    Calculates the parameters for the De Soto single diode model using the
+    procedure described in [1]. This procedure has the advantage of
+    using common specifications given by manufacturers in the
+    datasheets of PV modules.
+    The solution is found using the scipy.optimize.root() function,
```

Comment:
If you want this to be a separate paragraph then you'll need to add a blank line between them. Same for potential paragraph below. Either is fine with me. Now that you've added the line to the `api.rst` file, you can view the rendered documentation by clicking on the readthedocs link in the CI box at the bottom of the PR.